Description: Disable -Werror to avoid toolchain changes introducing FTBFS bugs
Author: Stephen Kitt <steve@sk2.org>

--- mingw-w64.orig/mingw-w64-tools/gendef/Makefile.am
+++ mingw-w64/mingw-w64-tools/gendef/Makefile.am
@@ -4,7 +4,7 @@
 
 gendef_SOURCES = src/gendef.h src/compat_string.h src/gendef.c src/gendef_def.c src/compat_string.c \
                  src/fsredir.c src/fsredir.h
-gendef_CFLAGS = $(AM_CFLAGS) -O3 -g -Werror -Wall -Wextra
+gendef_CFLAGS = $(AM_CFLAGS) -O3 -g -Wall -Wextra
 
 EXTRA_DIST = $(srcdir)/ChangeLog*
 DISTCHECK_CONFIGURE_FLAGS = --host=$(host) --with-mangle=@with_mangle@
--- mingw-w64.orig/mingw-w64-tools/genpeimg/Makefile.am
+++ mingw-w64/mingw-w64-tools/genpeimg/Makefile.am
@@ -1,6 +1,6 @@
 bin_PROGRAMS = genpeimg
 genpeimg_SOURCES = src/img.h src/img.c src/img_pe.c src/genpeimg.c
 genpeimg_CPPFLAGS = -Isrc
-genpeimg_CFLAGS = $(AM_CFLAGS) -O3 -g -Werror -Wall -Wextra
+genpeimg_CFLAGS = $(AM_CFLAGS) -O3 -g -Wall -Wextra
 
 DISTCHECK_CONFIGURE_FLAGS = --host=$(host)
